solidity/deploy/19_authorize_spv_maintainer_in_bridge.ts

```diff
@@ -0,0 +1,28 @@
+import { HardhatRuntimeEnvironment } from "hardhat/types"
+import { DeployFunction } from "hardhat-deploy/types"
+
+const func: DeployFunction = async function (hre: HardhatRuntimeEnvironment) {
+  const { getNamedAccounts, deployments } = hre
+  const { execute } = deployments
+  const { deployer, spvMaintainer } = await getNamedAccounts()
+
+  await execute(
+    "Bridge",
+    { from: deployer, log: true, waitConfirmations: 1 },
+    "setSpvMaintainerStatus",
+    spvMaintainer,
+    true
+  )
+}
+
+export default func
+
+func.tags = ["AuthorizeSpvMaintainer"]
+func.dependencies = ["Bridge"]
+
+// SPV maintainer can submit SPV proofs to the Bridge. We authorize spvMaintainer
+// account for Hardhat network (unit tests) and Goerli (testnet) but we DO NOT
+// want to authorize it for Mainnet deployment. SPV maintainer will be authorized
+// separately by the Governance when sweeping will be activated.
+func.skip = async (hre: HardhatRuntimeEnvironment): Promise<boolean> =>
+  hre.network.name === "mainnet"
```

Comment:
At this step, we already have the `MaintainerProxy` as an authorized SPV maintainer (step 13). Can we make it explicit in this comment? Second question: what SPV maintainer will be authorized when we activate sweep? Won't we handle all cases through `MainainerProxy`?

Reply:
Ok, thanks for the clarification.